covid_moonshot_ml/__init__.py

```diff
@@ -2,12 +2,12 @@
 
 # Add imports here
 # from .covid_moonshot_ml import *
-from . import data
-from . import datasets
-from . import docking
-from . import nn
-from . import schema
-from . import utils
+# from . import data
+# from . import datasets
+# from . import docking
+# from . import nn
+# from . import schema
+# from . import utils
```

Comment:
I'm going to uncomment and these and leave them for now, but they can be removed when the repo is split up if it's decided that they're not needed

Reply:
I think it makes sense in the interest of having this behave as a package (ie it allows you to run stuff like
```python
import covid_moonshot_ml
covid_moonshot_ml.docking.run_docking(...)
```
as opposed to having to manually import everything you want to use separately). this should be moot though once we split the package up into separate bits as you'll be able to use only the relevant package

Reply:
Oh I see. Yeah that would make sense if we wanted to keep covid_moonshot_ml as a combined package. I think we should remove it for now though since we know we don't want to do that?

Reply:
Fair enough, resolved in 52674c5.